plugins/inputs/win_eventlog/zsyscall_windows.go

```diff
@@ -0,0 +1,164 @@
+//+build windows
+
+//revive:disable-next-line:var-naming
+// Package win_eventlog Input plugin to collect Windows Event Log messages
+package win_eventlog
+
+import (
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+var _ unsafe.Pointer
+
+// EvtHandle uintptr
+type EvtHandle uintptr
+
+// Do the interface allocations only once for common
+// Errno values.
+const (
+	//revive:disable-next-line:var-naming
+	errnoERROR_IO_PENDING = 997
+)
+
+var (
+	//revive:disable-next-line:var-naming
+	errERROR_IO_PENDING error = syscall.Errno(errnoERROR_IO_PENDING)
+)
+
+// EvtFormatMessageFlag defines the values that specify the message string from
+// the event to format.
+type EvtFormatMessageFlag uint32
+
+// EVT_FORMAT_MESSAGE_FLAGS enumeration
+// https://msdn.microsoft.com/en-us/library/windows/desktop/aa385525(v=vs.85).aspx
+const (
+	//revive:disable:var-naming
+	// Format the event's message string.
+	EvtFormatMessageEvent EvtFormatMessageFlag = iota + 1
+	// Format the message string of the level specified in the event.
+	EvtFormatMessageLevel
+	// Format the message string of the task specified in the event.
+	EvtFormatMessageTask
+	// Format the message string of the task specified in the event.
+	EvtFormatMessageOpcode
+	// Format the message string of the keywords specified in the event. If the
+	// event specifies multiple keywords, the formatted string is a list of
+	// null-terminated strings. Increment through the strings until your pointer
+	// points past the end of the used buffer.
+	EvtFormatMessageKeyword
+	// Format the message string of the channel specified in the event.
+	EvtFormatMessageChannel
+	// Format the provider's message string.
+	EvtFormatMessageProvider
+	// Format the message string associated with a resource identifier. The
+	// provider's metadata contains the resource identifiers; the message
+	// compiler assigns a resource identifier to each string when it compiles
+	// the manifest.
+	//revive:disable-next-line:confusing-naming
+	EvtFormatMessageId
+	// Format all the message strings in the event. The formatted message is an
+	// XML string that contains the event details and the message strings.
+	EvtFormatMessageXml
```

Comment:
`EvtFormatMessageChannel`, `EvtFormatMessageProvider`, `EvtFormatMessageId` and `EvtFormatMessageXml` are unused.